### PR TITLE
updated grafana Dockerfile to use performance-dashboards repo

### PR DESCRIPTION
**Pull request by amitsagtani97**

### CODE REVIEW

File: containers/grafana/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Dockerfile for scale-ci-diagnosis
 
+# Stage-1 to generate the rendered json files from jsonnet templates.
+FROM registry.access.redhat.com/ubi8/ubi-minimal as jsonnet-builder
+
+RUN microdnf install git make tar
```
Comment (chaitanyaenr) [inline containers/grafana/Dockerfile]: We might want to combine the RUN statements to reduce the number of layers. Thoughts?

Comment (amitsagtani97) [reply containers/grafana/Dockerfile]: makes sense, updated.

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Comment by mohit-sheth** [comment]
LGTM

**Review by mohit-sheth — APPROVED** [review]

**Pull request merged**